examples/android/src/main/kotlin/com/connectrpc/examples/android/ElizaChatActivity.kt

```diff
@@ -156,15 +159,13 @@ class ElizaChatActivity : AppCompatActivity() {
             } catch (e: ConnectException) {
                 adapter.add(MessageData("Session failed with code ${e.code}", true))
             }
-            lifecycleScope.launch(Dispatchers.Main) {
-                buttonView.setOnClickListener {
-                    val sentence = editTextView.text.toString()
-                    adapter.add(MessageData(sentence, false))
-                    editTextView.setText("")
-                    // Send will be streaming a message to Eliza.
-                    lifecycleScope.launch(Dispatchers.IO) {
-                        stream.send(ConverseRequest.newBuilder().setSentence(sentence).build())
-                    }
+            buttonView.setOnClickListener {
```

Comment:
Should code that deals with the UI still be dispatched to Dispatchers.Main? I'm not familiar with android development but we might want to keep this here.

Reply:
The documentation for [`LifecycleCoroutineScope`](https://developer.android.com/reference/kotlin/androidx/lifecycle/LifecycleCoroutineScope) (the type returned by `lifecycleScope`) states that it is already tied to `Dispatchers.Main`. So the above call to `lifecycleScope.launch` should already dispatch in the right context.

I can set it explicitly above if you'd prefer. I wasn't really sure what was the most idiomatic for Android apps -- to omit when using the default or to always specify explicitly. I guess, even if it is idiomatic to omit, we might want to state it explicitly, just for maximum clarity.